src/emm/emm_cli.py

```diff
@@ -211,5 +225,25 @@ def list_modules(ctx: click.Context, enabled: bool, show_details: bool) -> None:
     orchestrator.display_modules(enabled, show_details)
 
 
+@cli.command(context_settings=dict(max_content_width=100, allow_extra_args=True))
+@click.option(
+    "--op",
+    "--operation",
+    type=click.Choice([a.value for a in GitAction]),
+    default=GitAction.CHECKOUT,
+    help="Git operations to perform with found commit [default=checkout].",
+)
+@click.option("-b", "--branch", default=None, help="Name of branch for git checkout.")
+@click.option("--directory", default=None, help="Directory to execute command at.")
+@click.pass_context
+def git(ctx: click.Context, op: str, branch: str, directory: str) -> None:
+    """Perform basic git checkout|rebase|merge operations to the specific revision."""
+    orchestrator = EmmOrchestrator()
+    for a in GitAction:
```

Comment:
You don't need to do this. First, you might want to change `GitAction` to a str-enum `class GitAction(str, Enum)`. Then you can make the argument type `op: GitAction` and python will do this conversion for you. You can see [here](https://github.com/dbradf/git-co-evg-base/blob/a2b2b5b437133141fe5dc9dadbdea74fde88c242/src/goodbase/goodbase_cli.py#L324) for an example.